salt/modules/snapper.py

```diff
@@ -0,0 +1,632 @@
+# -*- coding: utf-8 -*-
+'''
+Module to manage filesystem snapshots with snapper
+
+:codeauthor:    Duncan Mac-Vicar P. <dmacvicar@suse.de>
+:codeauthor:    Pablo Suárez Hernández <psuarezhernandez@suse.de>
+
+:depends:       ``dbus`` Python module.
+:depends:       ``snapper`` http://snapper.io, available in most distros
+:maturity:      new
+:platform:      Linux
+'''
+
+from __future__ import absolute_import
+
+import logging
+import os
+import time
+import difflib
+from pwd import getpwuid
+
+from salt.exceptions import CommandExecutionError
+import salt.utils
+
+
+try:
+    import dbus  # pylint: disable=wrong-import-order
+    HAS_DBUS = True
+except ImportError:
+    HAS_DBUS = False
+
+
+DBUS_STATUS_MAP = {
+    1: "created",
+    2: "deleted",
+    4: "type changed",
+    8: "modified",
+    16: "permission changed",
+    32: "owner changed",
+    64: "group changed",
+    128: "extended attributes changed",
+    256: "ACL info changed",
+}
+
+
+log = logging.getLogger(__name__)  # pylint: disable=invalid-name
+
+if HAS_DBUS:
+    bus = dbus.SystemBus()  # pylint: disable=invalid-name
+    snapper = dbus.Interface(bus.get_object('org.opensuse.Snapper',  # pylint: disable=invalid-name
```

Comment:
@cachedout That path is the snapper dbus interface. It will be the same in every distro, just like firewallD dbus path is `org.fedoraproject.FirewallD1` in every distro.